wasp/apps/calc.py

```diff
@@ -0,0 +1,1045 @@
+    b'\x08\x01@\xc1e\x01\x08\x80\x01\x81\xc05\xe8\x81\x0b\x01'
+    b'@4A\xc6A\x80\x08\x81\x12\xc0\x01\xc1A@5F'
+    b'\x804\x81\x01\x08\xc0\x81\xc1@\xd7I\x80\xdb\x81\x10\x01'
+    b'\xc0\xeb\xc1I@\x81A\x08\x01\x80\xc1\x8d\xc0\x9d\xc1\x08'
+    b'@$A\x8e\x01\t\x80*\x81\xc0\xc1\xe3\x81\n@\x08'
+    b'A\x805\xa6A\r\xc0\x01\xc1\x88@3A\xc1\x0e\x01'
+    b'\x80\x08\x81\xc05\xc1\xc7@\x01A\t\x01\x80\xfb\x81\xc0'
+    b'\xd7\xc9@\xebA\x80\xfc\x81\r\xc0\xfe\xc1@\xfbA\x80'
+    b'\xd7\x89A\x01\t\xc0$\xc1@\xc1L\x80\x9d\x81\x08\xc1'
+    b'M\xc1\x0b\xc0r\xc1a\xc1\x0b\x01@4A\x805\xa4'
+    b'A\x01\x0e\xc0\x08\xc1\x88\x81@3A\x80\x07\x81\x01\t'
+    b'\xc0\x01\xc1@\x08A\x804\x81\xc05\xc8A\x0b@\xfe'
+    b'A\x80\xd7\x8a\x81\xc0\x81\xc1@\xfdA\x01\x07\x01\x80\xff'
+    b'\x81\xc0\xeb\xc1@\xd7K\x80\xfe\x81\x0b\xc0N\xc1@\xc1'
+    b'K\x80\x9d\x81\x08\xc0$\xc1L@NA\x0c\x01\x80y'
+    b'\x81\xc0\xc1\xdf\x81\x01\x0c\x01@5d\x01\x10\x80\t\x81'
+    b'JA\xc03\xc1\x81@\x08AAAAA\xc1\x804'
+    b'\x81\xc05\xca@\tA\r\x80\xdb\x81\xc0\xd7\xcc@\xfb'
+    b'A\x80\xeb\x81\xc0\x81\xc1@VAA\x80\xdb\x81\xc1\xc0'
+    b'\xeb\xc1@\xd7AL\x81\r\x80r\x81\xc0\xc1\xca@\x9d'
+    b'A\x08\x80$\x81\xcb\xc0r\xc1\x0e\x01A@\xc1]\x80'
+    b'\x9d\x81\x01\x0e\xc0\x01\xc1@5b\xc1\x11\x01\x803\x81'
+    b'\\\x81\x01\x0e\xc0\x81\xc1@\xd7_\xc1\x0e\x01\x80r\x81'
+    b'\xc0\xc1\xc9@\x9dA\x08\x80$\x81\xca\xc0r\xc1\x01\x0f'
+    b'\x01@yA\x80\xc1\x9bA\x01\x10\xc0\x01\xc1@5`'
+    b'\xc1\x13\x01\x80\t\x81Z\x81\x01\x10\xc0\xeb\xc1@\xd7]'
+    b'\xc1\x10\x01\x80N\x81\xc0\xc1\xdb\x81\x01\x12\x81\xd9\x81\x13'
+    b'\x01@4A\x805\x9cA\x01\x16\xc0\x08\xc1\x98\xc1\x13'
+    b'@\x81A\x80\xd7\x9bA\x13\xc0N\xc1@\xc1Y\xc1\x15'
+    b'\x80$\x81\xc0\x9d\xc1U\xc1\x81\x15\x01@\x08A\x805'
+    b'\x9aA\x01\x18\xc0\x01\xc1@4A\x94A\xc1\x15\x80\xdb'
+    b'\x81\xc0\xd7\xd9\x81\x15@$A\x80\x9d\x81\xc0\xc1\xd5\x81'
+    b'A\x17\x01@*A\xc1\xd1\xc1A\x01\x18\x80\x01\x81\xc0'
+    b'4\xc1@5V\xc1\x81\x1b\x01\x80\x08\x81R\x81\x01\x17'
+    b'\xc0\xfe\xc1@\xacA\x80\xd7\x95A\xc1\x18\xc0*\xc1@'
+    b'\x9dA\x80\xc1\x91A\xc1\x1b\x01\xc1\xc0y\xc1\x8d\xc1@'
+    b'*A\x01\x1b\x01\x80\x07\x81\xc04\xc1@5R\xc1\x81'
+    b'\x01\x1e\x01\x81\xc1L\xc1\x81\x01\x1b\x80\xdb\x81\xc0\xfb\xc1'
+    b'@\xd7Q\xc1\x81\x1b\x01\x80$\x81\xc0r\xc1@\xc1M'
+    b'\xc1\x81 \x01\x81\x80N\x81\xc1\xc0\x9d\xc1\xc1A\xc1\xc1'
+    b'@rA\x81\x80$\x81\x01!\xc0\x01\xc1@\tA\x80'
+    b'5\x8eA\xc1$\x01\xc0\x07\xc1@\x08A\x803\x81\xc0'
+    b'4\xc1@5B\xc1\x81\x80\x08\x81\xc0\x07\xc1\x01 @'
+    b'VA\x80\xac\x81\xc0\xd7\xcd\x81A \x01@$A\x80'
+    b'N\x81\xc0r\xc1@\x9dAA\x80\xc1\x81AA\xc1\xc0'
+    b'N\xc1@$A\x01?\x13\x01\x80\x01\x81\xc0\x07\xc1@'
+    b'\tA\x803\x81\xc04\xc1@5B\xc1\x81\x80\t\x81'
+    b'\xc0\x07\xc1@\x01A\x01?\x15\x80\xfc\x81\xc0V\xc1@'
+    b'\x81A\x80\xac\x81\xc0\xfb\xc1@\xd7AAA\xc1\x81\x80'
+    b'\x81\x81\xc0V\xc1@\xfcA%\x01\x02\x02\x02\x01?\xff'
+    b'\xff\x19'
+)
+
+# 2-bit RLE, generated from res/calc.png, 413 bytes
```

Comment:
Given the app uses rounded buttons it seems a bit odd for the icon to use square buttons (and with a different colour scheme to the application)